specification/dartLangSpec.tex

```diff
@@ -14793,71 +14794,151 @@ \subsubsection{For-in}
 which is not a subtype of \code{Iterable<dynamic>}.
 }
 
+\LMHash{}%
+The scope introduced by \code{\{\,$S$\,\}} around $S$
+is known as the
+\IndexCustom{scope of the for-in statement $s$}{%
```

Comment:
Should this  be `$S$` ?

Reply:
We use `$s$` to denote a statement in several situations where it gets a name, and never `$S$` (that's mostly a type). So it wouldn't improve consistency to call it `$S$` here.